app/presenters/track.rb

```diff
@@ -15,31 +15,51 @@ def method_missing(method, *args)
       end
 
       def docs
-        track_docs = trackler_track.docs(image_path: "/api/v1/tracks/%s/images/docs/img" % @track_id)
-
-        track_docs.each_pair do |topic_name, topic_content|
-          track_docs[topic_name] = if topic_content.present?
-                                     topic_content
-                                   else
-                                     fallback_topic_content(topic_name)
-                                   end
+        trackler_topics.each_with_object(OpenStruct.new) do |topic_key, result|
+          result[topic_key] = topic_content(topic_key)
         end
-
-        track_docs
       end
 
       private
 
-      def fallback_topic_content(topic_name)
-        filepath = "./x/docs/md/track/#{topic_name.upcase}.md"
-        return '' unless File.exist?(filepath)
-        File.read(filepath)
-      end
-
       def trackler_track
         Trackler.tracks[@track_id]
       end
 
+      def trackler_docs
+        trackler_track.docs(image_path: "/api/v1/tracks/%s/images/docs/img" % @track_id)
+      end
+
+      def trackler_topics
+        trackler_docs.to_h.keys
+      end
+
+      def trackler_topic_content(topic_key)
+        trackler_docs[topic_key]
+      end
+
+      def topic_content(topic_key)
+        if trackler_topic_content(topic_key).present?
+          [trackler_topic_content(topic_key).strip, contributing_instructions(topic_key)].join("\n")
+        else
+          fallback_topic_content(topic_key)
+        end
+      end
+
+      def contributing_instructions(topic_key)
+        File.
+          read("./x/docs/md/track/BETTER.md").
+          gsub('REPO', trackler_track.repository).
```

Comment:
This commit had not been merged yet, so I assume that this was a pre-existing bug?